### DISCUSSION TITLE
Can't change away from AppleStock

### DISCUSSION THREAD

**Discussion by vexxvakan** [airbnb/visx#1343] [Q&A]
I copied the following CodeSandbox: [Here](https://codesandbox.io/s/autumn-currying-00l89)

Currently the chart is only working when its receiving AppleStock data, if I change the type of the getDate and getStockValue functions to my own Object called "WifiUser" and import it it doesnt work. The data inside of the AppleStock and WifiUser would be the same, I copied it straight over yet it fails me with "TypeError: Cannot read property 'date' of undefined" which makes no sense at all. I appreciate any kind of help since this is kind of nerve wrecking

**Comment by williaster** [comment]
can you link to the failing example? all you should need to to is
- update the accessor functions
- update the actual data that you pass throughout the example (replace `stock` and `filteredStock` with your data)
